bolt/lib/Profile/BoltAddressTranslation.cpp

```diff
@@ -114,9 +114,12 @@ void BoltAddressTranslation::write(const BinaryContext &BC, raw_ostream &OS) {
                       << Twine::utohexstr(Address) << ".\n");
     encodeULEB128(Address, OS);
     encodeULEB128(NumEntries, OS);
+    uint64_t InOffset = 0, OutOffset = 0;
```

Comment:
It's not an offset, right? Maybe a better name would be PrevIn, PrevOut? (as in "Previous Input Address")

Reply:
No, these are offsets, BAT.md is incorrect here